code/game/objects/objs.dm

```diff
@@ -63,6 +66,85 @@
 		visible_message("<span class='danger'>[src] shatters into a million pieces!</span>")
 		qdel(src)
 
+/obj/proc/start_pulling(atom/movable/AM,gs)

```

Comment:
If we need pulling on obj and mob, it should be an atom proc.

This *should* be it's own pr.

Reply:
I've tryed. And it worse. It broken lights. And mob still need its own pulling proc.
Also this proc is slightly different on mobs and objs. Sad,but not worked.

Reply:
You'll probably have to do this change first in another pr *then* come back to this one

Reply:
I've separated claw from everything else. I'll make it as another PR.